sections/footer.liquid

```diff
@@ -406,6 +408,9 @@
             "default": "left"
           }
         ]
+      },
+      {
+        "type": "@app"
```

Comment:
Maybe it's a case of the Monday mornings, but will "@app" match `{% when "app" %}` ?

Reply:
It will, which is a little surprising.